artifactory/1.11/install-postgres.md

```diff
@@ -0,0 +1,62 @@
+# Setting up Postgres on DC/OS for Artifactory
+
+## Prerequisites
+
+- DC/OS 1.11 or later with at least one public agent
+- [DC/OS CLI installed](https://dcos.io/docs/1.10/usage/cli/install/) and
+  configured to use your cluster
+
+## Setting up Postgres:
+
+1. Create a new file called `postgresql-options.json` with following content:
+
+```
+{
+  "service": {
+    "name": "postgresql"
+  },
+  "postgresql": {
+    "cpus": 0.5,
+    "mem": 512
+  },
+  "database": {
+    "username": "jfrogdcos",
+    "password": "jfrogdcos",
+    "dbname": "artifactory"
+  },
+  "storage": {
+    "host_volume": "/tmp",
+    "pgdata": "pgdata",
+    "persistence": {
+      "enable": false,
+      "volume_size": 512,
+      "external": {
+        "enable": false,
+        "volume_name": "postgresql",
+        "provider": "dvdi",
+        "driver": "rexray"
+      }
+    }
+  },
+  "networking": {
+    "port": 5432,
+    "host_mode": true,
+    "external_access": {
+      "enable": false,
+      "external_access_port": 15432
+    }
+  }
+}
+```
+
+2. Run this command to install Postgres:
+
+```
+dcos package install --options=postgresql-options.json postgresql
+```
+
+3. Make sure Postgres is running and is healthy by looking under the Services tab
+   in the DC/OS UI.
+
+This should be sufficient to trial the Artifactory package but we do not
+currently recommend using this for a production deployment of Artifactory.
```

Comment:
Could you give a hint here as to what you would want to use for production? A different database? Something that's HA or somehow distributed?

Reply:
postgresql-options.json mentioned in this page is a sample. So, we mention that it is not recommended for production deployment.
DB choice, CPU, memory and volume size needs to be picked and adjusted based on the expected load for the production Artifactory.

Reply:
Yes, *I* know this :smile: I was suggesting you add it to the README so it's clear.

Reply:
Ok.